src/fpm/manifest/profiles.f90

```diff
@@ -0,0 +1,778 @@
+              & new_profile('release', 'ifx', OS_ALL, flags = ' -fp-model=precise -pc64&
+                                                & -align all -error-limit 1 -reentrancy threaded&
+                                                & -nogen-interfaces -assume byterecl', is_built_in=.true.), &
+              & new_profile('release', 'ifx', OS_WINDOWS, flags = ' /fp:precise /align:all&
+                                                & /error-limit:1 /reentrancy:threaded&
+                                                & /nogen-interfaces /assume:byterecl', is_built_in=.true.), &
+              & new_profile('release', 'nagfor', OS_ALL, flags = ' -O4 -coarray=single -PIC', is_built_in=.true.), &
+              & new_profile('debug', 'caf', OS_ALL, flags = ' -Wall -Wextra -Wimplicit-interface&
+                                                & -fPIC -fmax-errors=1  -g -fcheck=bounds&
+                                                & -fcheck=array-temps -fbacktrace', is_built_in=.true.), &
+              & new_profile('debug', 'gfortran', OS_ALL, flags = ' -Wall -Wextra -Wimplicit-interface&
+                                                & -fPIC -fmax-errors=1 -g -fcheck=bounds&
+                                                & -fcheck=array-temps -fbacktrace -fcoarray=single', is_built_in=.true.), &
+              & new_profile('debug', 'f95', OS_ALL, flags = ' -Wall -Wextra -Wimplicit-interface&
+                                                & -fPIC -fmax-errors=1 -g -fcheck=bounds&
+                                                & -fcheck=array-temps -Wno-maybe-uninitialized&
+                                                & -Wno-uninitialized -fbacktrace', is_built_in=.true.), &
+              & new_profile('debug', 'nvfortran', OS_ALL, flags = ' -Minform=inform -Mbackslash -g&
+                                                & -Mbounds -Mchkptr -Mchkstk -traceback', is_built_in=.true.), &
+              & new_profile('debug', 'ifort', OS_ALL, flags = ' -warn all -check all -error-limit 1&
+                                                & -O0 -g -assume byterecl -traceback', is_built_in=.true.), &
+              & new_profile('debug', 'ifort', OS_WINDOWS, flags = ' /warn:all /check:all /error-limit:1&
+                                                & /Od /Z7 /assume:byterecl /traceback', is_built_in=.true.), &
+              & new_profile('debug', 'ifx', OS_ALL, flags = ' -warn all -check all -error-limit 1&
+                                                & -O0 -g -assume byterecl -traceback', is_built_in=.true.), &
+              & new_profile('debug', 'ifx', OS_WINDOWS, flags = ' /warn:all /check:all /error-limit:1&
+                                                & /Od /Z7 /assume:byterecl', is_built_in=.true.), &
+              & new_profile('debug', 'nagfor', OS_ALL, flags = ' -g -C=all -O0 -gline -coarray=single -PIC', is_built_in=.true.) &
+              &]
```

Comment:
Could you format this so it's a bit easier to read. I'd suggest a style like the following, but right now this is a bit difficult to grok quickly, so whatever can make it easier would be worth it.

```Fortran
default_profiles = &
    [ new_profile(
        'release', &
        'caf', &
        OS_ALL, &
        flags = '-O3 -Wimplicit-interface -fPIC -fmax-errors=1 -funroll-loops', &
        is_built_in = .true.) &
    , new_profile( &
        ...) &
    , ... &
    ]
```